pkgs/tools/filesystems/nixpart/0.4/default.nix

```diff
@@ -66,10 +67,11 @@ in buildPythonApplication rec {
 
   doCheck = false;
 
-  meta = {
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/aszlig/nixpart";
```

Comment:
```suggestion
    homepage = "https://github.com/NixOS/nixpart";
```

Reply:
but the src still points at aszlig/nixpart, should we change it too ?

Reply:
Good point :+1: